src/main/main.ts

```diff
@@ -291,10 +303,19 @@ ipcMain.on('settingsWindow', (event, args) => {
   }
     
   if (settingsWindow === null) return; 
-
+  
   if (args[0] === "quit") {
     console.log("User closed settings");
     settingsWindow.close();
+    storageDir = getStringConfigSafe(cfg, 'storage-path');
+    baseLogPath = getStringConfigSafe(cfg, 'log-path');
+    maxStorage = getStringConfigSafe(cfg, 'max-storage');
+    if (checkConfig()) {
+      updateStatus(0);
+      recorder = new Recorder(storageDir, maxStorage);  
+      watchLogs(String(baseLogPath));
```

Comment:
Why is String() needed here?

Reply:
leftover from a debugging session where i was getting an error on this. i doubt its needed now. good catch.